### PR TITLE
Ashen

### PR DESCRIPTION
**Pull request by Cojomax99**

### CODE REVIEW

File: src/main/java/net/tropicraft/core/common/entity/damage/DartDamage.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package net.tropicraft.core.common.entity.damage;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.util.EntityDamageSourceIndirect;
+
+public class DartDamage extends EntityDamageSourceIndirect {
```
Comment (tterrag1098) [inline src/main/java/net/tropicraft/core/common/entity/damage/DartDamage.java]: What's the point of this class?

File: src/main/java/net/tropicraft/core/registry/EntityRenderRegistry.java [hunk 2]
```diff
@@ -73,8 +76,10 @@ public static void init() {
 		registerEntityRender(EntityChair.class, new RenderChair());
 		registerEntityRender(EntityUmbrella.class, new RenderUmbrella());
 		registerEntityRender(EntityCoconutGrenade.class, new RenderSnowball(Minecraft.getMinecraft().getRenderManager(),
-				ItemRegistry.coconutBomb,
-				Minecraft.getMinecraft().getRenderItem()));
+																			ItemRegistry.coconutBomb,
```
Comment (tterrag1098) [inline src/main/java/net/tropicraft/core/registry/EntityRenderRegistry.java]: That indent tho

**Review by tterrag1098 — DISMISSED** [review]

File: src/main/java/net/tropicraft/core/common/entity/damage/DartDamage.java (hunk 1 above)
Comment (Cojomax99) [reply src/main/java/net/tropicraft/core/common/entity/damage/DartDamage.java]: oh this was used when I had darts then I removed them...my b

File: src/main/java/net/tropicraft/core/registry/EntityRenderRegistry.java (hunk 2 above)
Comment (Cojomax99) [reply src/main/java/net/tropicraft/core/registry/EntityRenderRegistry.java]: .....YOU SEE NOTHING

**Pull request merged**